packages/grid/src/styled/splitter/StyledPane.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { DEFAULT_THEME, retrieveComponentStyles } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'pane';
+
+export const StyledPane = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  position: relative;
+  min-width: 0;
+  min-height: 0;
```

Comment:
Just wondering, what is the reason for setting the min-*dimension to `0`?

Reply:
@jzempel worked on this part. I will defer to him to answer this.

Reply:
The assumption of this component is that a `Pane` is participating as a child of a grid or flex layout. This overrides the [default](https://developer.mozilla.org/en-US/docs/Web/CSS/min-width#formal_definition) of `auto` which prevents the child's ability to fully collapse – depending on the `Pane.Splitter`'s `min` prop.